manager/controllers/app/blueprint_controller.go

```diff
@@ -141,10 +141,14 @@ func (r *BlueprintReconciler) deleteExternalResources(blueprint *app.Blueprint)
 	return errors.New(strings.Join(errs, "; "))
 }
 
-func (r *BlueprintReconciler) applyChartResource(log logr.Logger, ref string, vals map[string]interface{}, kubeNamespace string, releaseName string) (ctrl.Result, error) {
-	log.Info(fmt.Sprintf("--- Chart Ref ---\n\n%v\n\n", ref))
+func (r *BlueprintReconciler) applyChartResource(ctx context.Context, log logr.Logger, chartSpec app.ChartSpec, args map[string]interface{}, kubeNamespace string, releaseName string) (ctrl.Result, error) {
+	log.Info(fmt.Sprintf("--- Chart Ref ---\n\n%v\n\n", chartSpec.Name))
 
-	nbytes, _ := yaml.Marshal(vals)
+	args = CopyMap(args)
```

Comment:
Wouldn't an explicit new variable be better here?

Reply:
no, because it gives no value to helping understand the code